ui-v2/tests/acceptance/components/catalog-filter.feature

```diff
@@ -81,8 +81,39 @@ Feature: components / catalog-filter
     ---
     And I see 1 [Model] model
     And I see 1 [Model] model with the port "65535"
+    Then I fill in with yaml
+    ---
+    s: service-0-with-id
+    ---
+    And I see 1 [Model] model
+    And I see 1 [Model] model with the id "service-0-with-id"
```

Comment:
Is this ID hardcoded in the API double?

Reply:
Yup its set in the fixtures/double:

https://github.com/hashicorp/consul-api-double/blob/12b36818eb637372ea44d29e1b22e325cc19d002/v1/internal/ui/node/_#L17

https://github.com/hashicorp/consul-api-double/blob/12b36818eb637372ea44d29e1b22e325cc19d002/v1/health/service/_#L22